packages/sentry/src/index.ts

```diff
@@ -38,8 +34,4 @@ export const initSentry: (
       "SecureTunnelInactiveForAppUrlError",
     ],
   });
-
-  addExtensionMethods();
```

Comment:
Do we need to register tracing when initialising Sentry now that this is dropped? https://github.com/getsentry/sentry-javascript/blob/7.47.0/MIGRATION.md/#node

Reply:
Ah probably. Let me add that in.

Reply:
Ah wait, this was removed in v9 actually and now we don't need anything at all: https://docs.sentry.io/platforms/javascript/guides/hapi/migration/v7-to-v8/#updated-sdk-initialization